compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -454,17 +454,18 @@ object Flags {
   /** Flags representing access rights */
   final val AccessFlags = Private | Protected | Local
 
-  /** Flags guaranteed to be set upon symbol creation */
+  /** Flags that are not (re)set when completing the denotation */
```

Comment:
I think both the old and the new comment are useful. Can we combine them?

Reply:
I think the old comment is misleading. These flags are not guaranteed to be set up on symbol creation, as can be witnessed by them appearing in setFlag/resetFlag calls in various places. I think that correctness only requires them to not appear in completers and InfoTransformers, but they're free to appear in other places such as TreeTransformers (e.g, the `setFlag(Mutable)` in Memoize I mentioned above).